src/apps/relay/turn_ports.c

```diff
@@ -172,7 +172,7 @@ int turnports_allocate(turnports *tp) {
 
     while (1) {
 
-      if (tp->high <= tp->low) {
+      if ((tp->high - tp->low) == 0) {
```

Comment:
Explain how this is going to fix the problem?

This formulation is basically the same as use the `==` comparison operator, which is already covered by checking if `high` is less or equal to `low`.

Reply:
Better would be for this algorithm to have a proper circular queue of available port numbers. As a real data structure.

But the C-language makes it very difficult to encapsulate logic of that nature.